### PR TITLE
(OraklNode) Websocket feed data buffer preparation for 100+ more price pairs

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

Increase websocket feed data buffersize for 300% increased price pairs

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [x] Should publish Docker image

### CODE REVIEW

**Review by Intizar-T — APPROVED** [review]
lgtm!

File: node/pkg/websocketfetcher/app.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ import (
 
 const (
 	DefaultStoreInterval = 250 * time.Millisecond
-	DefaultBufferSize    = 500
+	DefaultBufferSize    = 3000
```
[the remark this answers is not in the document]
Comment (Intizar-T) [reply node/pkg/websocketfetcher/app.go:41]: interesting point! 🤔

[the remark this answers is not in the document]
Comment (nick-bisonai) [reply node/pkg/websocketfetcher/app.go:41]: each buffer has different speed and amount of data inflow the low level, fast data inflow buffer has larger buffer size

[the remark this answers is not in the document]
Comment (nick-bisonai) [reply node/pkg/websocketfetcher/app.go:41]: or should I make it all default buffer sizes into 3000 over multiple packages

**Pull request merged**